src/components/VolunteerForm/Fields/Input.js

```diff
@@ -1,51 +1,53 @@
 import React from "react";
 import { Form, Input, DatePicker } from "antd";
 import { toNumber } from "../utils";
+import { useTranslation } from "react-i18next";
 const { TextArea } = Input;
 
-export const MobileField = ({
-  label = "Mobile Number",
-  name = "mobile",
-  placeholder = "Enter your mobile number",
-}) => (
-  <Form.Item
-    label={label}
-    name={name}
-    rules={[
-      {
-        required: true,
-        message: "Mobile number is required",
-      },
-      {
-        type: "number",
-        transform: toNumber,
-        message: "Invalid number format",
-      },
-      {
-        len: 10,
-        message: "Mobile number must be 10 digits",
-      },
-    ]}
-  >
-    <Input maxLength={10} placeholder={placeholder} />
-  </Form.Item>
-);
+export const MobileField = () => {
+  const [t] = useTranslation();
+  return(
+    <Form.Item
+      label={t("mobile_number")}
```

Comment:
to better organise the translation, can all form related attributes be moved under a form key
form.label.mobile_number
form.validation_msg.mobile_number 
something like this?

Reply:
Sure i will change the structure as per above suggesstion